DTS-STN/Service-Canada-Labs
# Benefits Navigator Overview Page + Card on Home

# Benefits Navigator Overview Page + Card on Home

This task creates the benefits navigator overview page on the site. English and french paths working with applied rewrite. Please verify it's design and behaviour by checking out the designs [here](https://www.figma.com/file/ke3SrZNGMP3AjwzMMuEgbo/%5Bcurrent%5D-BDM-DECD-SC-Labs?type=design&node-id=15727-42759&mode=design&t=QRsJpdg3oyEAARFy-4)

![Screenshot 2023-07-21 at 3 00 44 PM](https://github.com/DTS-STN/Service-Canada-Labs/assets/31868510/5ebd3f9b-f094-4deb-a35e-a5e594a8ea85)

![Screenshot 2023-07-21 at 2 59 38 PM](https://github.com/DTS-STN/Service-Canada-Labs/assets/31868510/7c0f33ce-005b-4d4b-a75b-1b75c46a0906)

![Screenshot 2023-07-21 at 3 00 08 PM](https://github.com/DTS-STN/Service-Canada-Labs/assets/31868510/4dca0dbe-e183-4be6-ad99-f0778728c4ec)


## Test Instructions

1. Go to homepage and see that Benefits Navigator card is available
2. Click the card and navigate to the page
3. See that page matches the [design](https://www.figma.com/file/ke3SrZNGMP3AjwzMMuEgbo/%5Bcurrent%5D-BDM-DECD-SC-Labs?type=design&node-id=15727-42759&mode=design&t=QRsJpdg3oyEAARFy-4)
4. Toggle languages during navigation

## Definition of Done
